film_database/main.js

```diff
@@ -0,0 +1,215 @@
+// Favorite Show Section
+const favoriteShowSectionElement = document.getElementById("favorite-show-section")
+const favoriteShowInputElement = document.getElementById("favorite-show");
+const favoriteShowBtn = document.getElementById("favorite-show-btn");
+const h2FavoriteShowElement = document.getElementById("h2-favorite-show");
+
+// Discover New Show
+const newShowSectionElement = document.getElementById("new-show-section")
+const newShowBtn = document.getElementById("new-show-btn");
+const h2NewShowElement = document.getElementById("h2-new-show");
+
+//reset button
+const resetBtn = document.getElementById("reset-btn");
+
+// url source
+const urlSearchShowText = "https://api.tvmaze.com/search/shows?q=";
```

Comment:
Const variable declarations should be all caps (as in line 9). Source: [Google JavaScript Style guide](https://google.github.io/styleguide/javascriptguide.xml)

Reply:
Replaced "urlSearchShowText" to "URL_SEARCH_SHOW_TEXT"